ulog/context.go

```diff
@@ -74,8 +74,7 @@ func (t trace) MarshalLogObject(enc zapcore.ObjectEncoder) error {
 	if !j.IsValid() {
 		return fmt.Errorf("invalid span: %v", j.SpanID())
 	}
-	enc.AddUint64("span", j.SpanID())
-	enc.AddUint64("trace", j.TraceID())
-	enc.AddUint64("parent", j.ParentID())
```

Comment:
Note that, AFAIK, this will be quite a bit more expensive to index. If you intend these IDs to be useful for searching ElasticSearch, ints may perform better.